src/test/scala/io/iohk/ethereum/blockchain/sync/regular/RegularSyncSpec.scala

```diff
@@ -47,13 +47,13 @@ class RegularSyncSpec
   type Fixture = RegularSyncFixture
 
   val actorSystemResource =
-    Resource.make(Task { ActorSystem() })(system => Task { TestKit.shutdownActorSystem(system) })
+    Resource.make(Task { ActorSystem("default", classLoader = Some(getClass.getClassLoader)) })(system => Task { TestKit.shutdownActorSystem(system) })
   val fixtureResource = actorSystemResource.map(new Fixture(_))
 
   // Used only in sync tests
   var testSystem: ActorSystem = _
   override def beforeEach: Unit =
-    testSystem = ActorSystem()
+    testSystem = ActorSystem("default", classLoader = Some(getClass.getClassLoader))
```

Comment:
It looks a bit strange that we need 2 actor systems for one test and we use a non-standard way to close them. Normally we sth like:

```scala
class DebugServiceSpec
    extends TestKit(ActorSystem("ActorSystem_DebugServiceSpec"))
    with AnyFlatSpecLike
    with WithActorSystemShutDown
```

Maybe the issue arise, because we do not close properly ActorSystem and there are issues because of that?

Reply:
>Maybe the issue arise, because we do not close properly ActorSystem and there are issues because of that?

Hmm, probably, if an ActorSystem wasn't properly closed `afterEach` test, we'd have errors on a first run, but I have no errors at all there.